Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -396,11 +397,38 @@ function Invoke-CodeGen {
     Invoke-Task -name $MyInvocation.MyCommand.Name -task {
         $tool = (Join-Path $toolsPath 'EdFi.Ods.CodeGen')
         $repositoryRoot = (Get-RepositoryRoot $implementationRepo).Replace($implementationRepo, '')
-
-        if ($IncludePlugins) {
+        $total=$ExtensionPaths.Split(" ").Length
+        $counter=0
+
+        if (-not [string]::IsNullOrWhiteSpace($ExtensionPaths)) {
+            $counter=1
+            Foreach ($extensionPath in $ExtensionPaths.Split(" ")) {
+
+                if (![System.IO.Directory]::Exists($extensionPath)) {
+                    throw "Unable to find extension Location project path at location $extensionPath ."
+                }
+
+                if($counter -eq $total){
+                    $extensionLocationPlugins += $extensionPath
+                }
+                else
+                {
+                    $extensionLocationPlugins += $extensionPath + ','
+                }
+                $counter++
+            }
+        }
```

Comment:
if the `$ExtensionPaths` parameter is an array of strings I'm not sure that you will need any of this.

Reply:
I changed it . Thanks